mlkem/kem.c

```diff
@@ -105,13 +157,12 @@ int crypto_kem_enc_derand(uint8_t *ct, uint8_t *ss, const uint8_t *pk,
  *              - const uint8_t *pk: pointer to input public key
  *                (an already allocated array of MLKEM_PUBLICKEYBYTES bytes)
  *
- * Returns 0 (success)
+ * Returns 0 on success, and -1 if the public key modulus check fails
```

Comment:
Seeing that this is the top-level API, add reference to the relevant section of the FIPS203 spec?